odl/tomo/geometry/conebeam.py

```diff
@@ -307,6 +334,11 @@ def det_radius(self):
         """Detector circle radius of this geometry."""
         return self.__det_radius
 
+    @property
+    def det_curve_radius(self):
```

Comment:
Last-minute comment, but part of API, thus important. I think it's better to use "curvature" instead of "curve" since the description of `det_radius` is described as "Detector circle radius", and this one as "Detector curve radius". That can be very confusing since a circle is a curve.

So please change `det_curve_radius` everywhere to `det_curvature_radius`. It is a bit long but much clearer, and also valid for 2D detectors.

After that I'll merge, I promise :crossed_fingers: